Valet/VALValet.m

```diff
@@ -237,37 +237,35 @@ + (nullable NSString *)_sharedAccessGroupPrefix;
 
 #pragma mark - Initialization
 
-- (nullable instancetype)initWithIdentifier:(nonnull NSString *)identifier accessibility:(VALAccessibility)accessibility;
+- (nonnull instancetype)initWithIdentifier:(nonnull NSString *)identifier accessibility:(VALAccessibility)accessibility;
 {
     VALCheckCondition(identifier.length > 0, nil, @"Valet requires an identifier");
     VALCheckCondition(accessibility > 0, nil, @"Valet requires a valid accessibility setting");
     
     self = [super init];
-    if (self != nil) {
-        _baseQuery = [[self class] mutableBaseQueryWithIdentifier:identifier accessibility:accessibility initializer:_cmd];
-        _identifier = [identifier copy];
-        _sharedAcrossApplications = NO;
-        _accessibility = accessibility;
-        _lockForSetAndRemoveOperations = [NSLock new];
-    }
+
+    _baseQuery = [[self class] mutableBaseQueryWithIdentifier:identifier accessibility:accessibility initializer:_cmd];
+    _identifier = [identifier copy];
+    _sharedAcrossApplications = NO;
+    _accessibility = accessibility;
+    _lockForSetAndRemoveOperations = [NSLock new];
     
     return [[self class] sharedValetForValet:self];
 }
 
-- (nullable instancetype)initWithSharedAccessGroupIdentifier:(nonnull NSString *)sharedAccessGroupIdentifier accessibility:(VALAccessibility)accessibility;
+- (nonnull instancetype)initWithSharedAccessGroupIdentifier:(nonnull NSString *)sharedAccessGroupIdentifier accessibility:(VALAccessibility)accessibility;
 {
     VALCheckCondition(sharedAccessGroupIdentifier.length > 0, nil, @"Valet requires a sharedAccessGroupIdentifier");
     VALCheckCondition(accessibility > 0, nil, @"Valet requires a valid accessibility setting");
     
     self = [super init];
-    if (self != nil) {
-        _baseQuery = [[self class] mutableBaseQueryWithSharedAccessGroupIdentifier:sharedAccessGroupIdentifier accessibility:accessibility initializer:_cmd];
-        
-        _identifier = [sharedAccessGroupIdentifier copy];
-        _sharedAcrossApplications = YES;
-        _accessibility = accessibility;
-        _lockForSetAndRemoveOperations = [NSLock new];
-    }
+
```

Comment:
👍 

I'm a big fan of getting rid of `if (self != nil)` checks where they aren't needed.